skia/BUILD.gn

```diff
@@ -564,7 +564,7 @@ source_set("skia_opts") {
     if (is_linux || is_mac) {
       cflags += [ "-msse4.1" ]
     }
-  } else if (current_cpu == "arm") {
+  } else if (current_cpu == "arm" || current_cpu == "arm64") {
```

Comment:
this change seems a little dodgier - is it needed? i could imagine -fomit-frame-pointer being meaningful but the arm neon checks almost certainly don't apply in arm64